integration_tests/go.mod

```diff
@@ -119,3 +119,5 @@ replace (
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
 	github.com/tikv/client-go/v2 => ../
 )
+
+replace github.com/pingcap/tidb => github.com/windtalker/tidb v1.1.0-beta.0.20231020063218-4d1c15539f3f
```

Comment:
Why need this?

Reply:
Because there is a cycle dependency between tidb and client-go integration test. I change the interface of `Cluster`, intergration test will fail, I have use a local tidb to make the test pass.